index.html

```diff
@@ -0,0 +1,532 @@
+                            <div class="frame43-card-works-right-block-ruby-tag">
+                                <div class="frame43-card-works-right-block-tag">
+                                    <p class="works-tag-text">Ruby</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="frame43-card-works-action">
+                        <div class="frame43-card-works-action-action"></div>
+                        <button class="see-project"><p class="see-project-text">See Project</p></button>                        
+                    </div>
+                </div>
+                <div class="frame43-card-works1 card-work">
+                    <div class="frame43-card-works-mask-group">
+                        <div class="card-works-img1"></div>
+                    </div>
+                    
+                    <div class="frame43-card-works-right-block">
+                        <h3 class="frame43-card-works-right-block-title-post"><p id="works-title-text">Profesional Art Printing Data</p></h3>
+
+                        <p class="frame43-card-works-right-block-supporting-text">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. has been the industry's standard
+                        </p>
+
+                        <div class="frame43-card-works-right-block-tags">
+                            <div class="frame43-card-works-right-block-html-tag">
+                                <div class="frame43-card-works-right-block-tag">
+                                    <p class="works-tag-text">html</p>
+                                </div>
+                            </div>
+    
+                            <div class="frame43-card-works-right-block-bootstrap-tag">
+                                <div class="frame43-card-works-right-block-tag">
+                                    <p class="works-tag-text">bootstrap</p>
+                                </div>
+                            </div>
+    
+                            <div class="frame43-card-works-right-block-ruby-tag">
+                                <div class="frame43-card-works-right-block-tag">
+                                    <p class="works-tag-text">Ruby</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="frame43-card-works-action">
+                        <div class="frame43-card-works-action-action"></div>
+                        <button class="see-project"><p class="see-project-text">See Project</p></button>                        
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- About Me -->
+    <section id="about">
+        <h1 id="about-me">About me</h1>
```

Comment:
[OPTIONAL]

- I will recommend you change this to h2 tag, as it is recommended to have just a single h1 tag in a HTML file for semantic reasons.